### PR TITLE
Display alias in man page

### LINKED ISSUE #3353: List subcommand aliases on man page
When you run `wp help some command` it would be nice if the man page that is displayed would include any aliases for that command as well.

Eg: `wp help plugin update` would also show its alias `upgrade`.  
Perhaps something like this?

```
NAME

  wp plugin update

ALIASES

  wp plugin upgrade

DESCRIPTION
  ...
```

### PR DESCRIPTION
**Pull request by ernilambar**
Fixes #3353

### CODE REVIEW

**Comment by ernilambar** [comment]
Screenshot http://prnt.sc/ccu25c

File: php/commands/help.php [hunk 1]
```diff
@@ -58,6 +58,12 @@ private static function find_subcommand( $args ) {
 	private static function show_help( $command ) {
 		$out = self::get_initial_markdown( $command );
 
+		$alias = $command->get_alias();
```
Comment (danielbachhuber) [inline php/commands/help.php]: I think it makes more sense to put this in the `get_initial_markdown` method

File: php/commands/help.php [hunk 2]
```diff
@@ -121,6 +121,11 @@ private static function get_initial_markdown( $command ) {
 
 		$binding['synopsis'] = wordwrap( "$name " . $command->get_synopsis(), 79 );
 
+		$alias = $command->get_alias();
+		if ( $alias ) {
+			$binding['has-alias']['alias'] = $alias;
```
Comment (danielbachhuber) [inline php/commands/help.php]: This can be `$binding['alias']`

File: templates/man.mustache [hunk 3]
```diff
@@ -12,6 +12,12 @@
 
 	{{synopsis}}
 
+{{#has-alias}}
```
Comment (danielbachhuber) [inline templates/man.mustache]: This can be `{{#alias}}`

The `#` in Mustache is an operator for checking whether the variable is set.

**Comment by danielbachhuber** [comment]
@ernilambar Can we get a test in here too?

**Pull request merged**